corehq/apps/hqwebapp/templatetags/proptable_tags.py

```diff
@@ -102,15 +107,19 @@ def _key_format(k, v):
             fmt = USER_DATE_FORMAT
 
         iso = val.isoformat()
-        ret = mark_safe("<time %s title='%s' datetime='%s'>%s</time>" % (
-            "class='timeago'" if timeago else "", iso, iso, safe_strftime(val, fmt)))
+        ret = format_html(
+            "<time {} title='{}' datetime='{}'>{}</time>",
+            "class='timeago'" if timeago else "",
```

Comment:
Should this be treated like the one above with `mark_safe("class='well'")`?
```suggestion
            mark_safe("class='timeago'") if timeago else "",  # nosec: no user input
```